clmm/theory/parent_class.py

```diff
@@ -374,3 +376,34 @@ def _eval_magnification(self, r_proj, z_cl, z_src):
         kappa = self.eval_convergence(r_proj, z_cl, z_src)
         gamma_t = self.eval_tangential_shear(r_proj, z_cl, z_src)
         return 1./((1-kappa)**2-abs(gamma_t)**2)
+    
+    def eval_magnification_bias(self, r_proj, z_cl, z_src, alpha):
```

Comment:
can you add an validation function? just follow the model of `eval_magnification`